### PR TITLE
League session details

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add some details for the league sessions, especially livestream details.

- Add livestream details for all sessions
- Populate time for 2nd session (12pm)
- Add code submission links

Thumbnails for all streams still show module 1. I'll update those soon.

The livestreams for sessions 3 and 4 are currently scheduled for 12pm too, but those can be moved should we need to.

### CODE REVIEW

File: _events/sr2021/league-2.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+date: 2021-02-06 12:00:00 Europe/London
 layout: event
 type: competition
-location: Student Robotics' YouTube Channel
+location: <a href="https://www.youtube.com/watch?v=RwW5Oz30gbE">Student Robotics' YouTube Channel</a>
 ---
 
 This is the second league round of this year's competition and we'll get to see what lessons teams have learnt from their first time in the arena! Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 8pm GMT on Friday 5th: [Submit your code](https://studentrobotics.org/code-submitter/)
```
Comment (sedders123) [inline _events/sr2021/league-2.md:11]: Wonder if we can reword this to be for a target audience of the general public, while still useful to teams that find the  the page?

```suggestion
Teams will have to submit their code to us by 8pm GMT on Friday 5th using our [code submitter](https://studentrobotics.org/code-submitter/)
```

File: _events/sr2021/league-2.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
 ---
 title: Competition League Session 2
-date: 2021-02-06
-time_tbc: true
+date: 2021-02-06 12:00:00 Europe/London
 layout: event
 type: competition
-location: Student Robotics' YouTube Channel
+location: <a href="https://www.youtube.com/watch?v=RwW5Oz30gbE">Student Robotics' YouTube Channel</a>
```
Comment (sedders123) [inline _events/sr2021/league-2.md:6]: Didn't know this worked, great idea!

File: _events/sr2021/league-2.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
 ---
 
 This is the second league round of this year's competition and we'll get to see what lessons teams have learnt from their first time in the arena! Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 8pm GMT on Friday 5th: [Submit your code](https://studentrobotics.org/code-submitter/)
+
+Our live-stream will start at noon GMT: [Watch live](https://www.youtube.com/watch?v=RwW5Oz30gbE)
```
Comment (sedders123) [inline _events/sr2021/league-2.md:13]: I've been trying to reword this to make it more casual (this was the best I got)
```suggestion
We'll be kicking things off at noon GMT, be sure to [watch us live](https://www.youtube.com/watch?v=RwW5Oz30gbE)!
```
but then I was wondering if we need this sentence at all? Both the time and the stream (embedded) will already be on the page?

**Review by sedders123 — COMMENTED** [review]
Thanks for putting this together, left a couple of thoughts inline 🙂

File: _events/sr2021/league-3.md [hunk 4]
```diff
@@ -4,7 +4,20 @@ date: 2021-03-20
...
 type: competition
-location: Student Robotics' YouTube Channel
+location: <a href="https://www.youtube.com/watch?v=MpuhtW4mCKM">Student Robotics' YouTube Channel</a>
 ---
 
 This is the third league round of this year's competition and we'll be able to see the result of a few months of robot iteration! Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 8pm UK time on Friday 19th: [Submit your code](https://studentrobotics.org/code-submitter/)
+
+Our live-stream will start at noon UK time: [Watch live](https://www.youtube.com/watch?v=MpuhtW4mCKM)
```
Comment (PeterJCLaw) [inline _events/sr2021/league-3.md:14]: Please could we be specific about the timezone here? If it's not GMT we should say explicitly that it's BST. People outside the UK (even in Europe) likely won't know exactly when our clocks change.

Even better would be to use a `<time>` element with a suitable `datetime` attribute to specify the instant precisely. It _may_ be worth extending our event-times JS stuff to being sufficiently general to add a dynamic title to such `<time>` elements, though maybe not worth it for now? (I suspect that the limitations of the translation handling maybe such that  in the context of prose having it say "8pm BST" for everyone is preferable to trying to swap in something location specific, but that may be worth exploring if we have time).

**Review by PeterJCLaw — APPROVED** [review]
Note that the streams themselves currently all have placeholder images which say "Session 1". We should fix that before merging (or very shortly afterwards).

File: _events/sr2021/league-2.md [hunk 5]
```diff
@@ -1,10 +1,20 @@
 ---
 title: Competition League Session 2
-date: 2021-02-06
-time_tbc: true
+date: 2021-02-06 12:00:00 Europe/London
```
Comment (PeterJCLaw) [inline _events/sr2021/league-2.md:3]: oh, small thing: If we're committing to noon for all of these, should we update that too now?
